db/db.js

```diff
@@ -15,6 +15,28 @@ const connect = async () => {
 }
 connect();
 
+
+// .then(() => sequelize.query('createdb potentialsound -O root'))
```

Comment:
remove comments if they are unused, if they refer to future code to be written, include the 'TODO:' tag ahead of them